### PR TITLE
Thieves Den Kit

### PR DESCRIPTION
**Pull request by liltenhead**
## About the PR
Adds the den kit as an option for the thief antagonist to select. 

## Why / Balance
Main reason to be added is for fun. This allows thieves to dismantle steel walls and replace them with wall doors in order to make a hidden hideout for their stolen goods. You could manually find every piece of this kit if you loot and steel enough, but this gives thieves the option of building a hideout without having to draw extra suspicion of acquiring the materials/tools in the first place. So it is a "weak" option but none of the items in this set make you "valid". 

## Media

den
tools

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added the thieves den kit.

### CODE REVIEW

File: Resources/Locale/en-US/thief/backpack.ftl [hunk 1]
```diff
@@ -61,3 +61,10 @@ thief-backpack-category-smuggler-description =
     and an invisible crate. You can't move in them,
     but you can quickly hide or carry valuable loot.
     This kit also has a cool void cloak to go along with it.
+
+thief-backpack-category-den-name = denmakers's kit
+thief-backpack-category-den-description =
+    A kit for those who want to live in the walls.
+    Includes a stack of steel, rods, cables, and
+    two powercells to make hidden doors.
+    Also includes the necessary saftey equipment.
```
Comment (asperger-sind) [inline Resources/Locale/en-US/thief/backpack.ftl:70]: **Saftey** equipment looks like a typo.

Comment (liltenhead) [reply Resources/Locale/en-US/thief/backpack.ftl]: spelling is hard

**Comment by Ilya246** [comment]
should have a full belt and insuls maybe

**Comment by TheShuEd** [comment]
Perhaps this set should replace the bugbear set, since this one is quieter.  But in my opinion, it needs to be made stronger

**Comment by TheShuEd** [comment]
Actually yes, the kit is almost useless: everything in it can be obtained or stolen almost without problems anyway

**Comment by liltenhead** [comment]
Yeah wasn't really meant to be op, it's just meant to allow for building a hideout without having to spend a bunch of time getting tools and materials from across the station. I can add a full toolbelt so you can dismantle/hack doors too.

**Comment by TheShuEd** [comment]
I propose to merge this with the existing set by removing C4
or add a scram implant, which would be a good fit for a thief.

but maybe wait and see what other people say.

**Comment by icekot8** [comment]
a paranoid man's kit. Too weak, useless kit, really. Hardly anyone would notice the lack of welding and crowbar :trollface:.
 I think we should add an EMP implant and a couple things besides it, like a light implant or a regular syndicate flash flashlight.

**Pull request closed without merging**